test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13098,6 +13098,10 @@ class MoM(PBSService):
                        access or dictionary containing
                        {'dbname':...,'user':...,'port':...}
     :type db_access: str or dictionary
+    :param revert_to_default: set to True if you like this instance of
+                              mom to be reverted to its default when
+                              revert_mom() is called.
+    :type revert_to_default: bool
```

Comment:
revert_to_defaults was not added as a param.  It is just set to true.  Do you really need it?

Reply:
You're right. It's not a param since it's not even passed to class MoM's init(). I just needed an instance variable that can be update per testsuite/test case. It defaults to True, but on the pbs_cgroups_hook ptl testsuite, I need to set it to False under setUp, so that the revert_to_defaults() mom action would not do the enable pbs_cgroups hook thing. I'll update the fix.

Reply:
Maybe move this comment to where you set the instance variable?

Reply:
Good idea. Thanks.